deployments/helm/nvidia-dra-driver-gpu/values.yaml

```diff
@@ -22,11 +22,6 @@
 # For driver installed directly on a host, a value of `/` is used.
 nvidiaDriverRoot: /
 
-# Specify the path of CTK binary (nvidia-ctk) on the host,
```

Comment:
If this option is not defined in the values file, does setting it still have an effect?